### DISCUSSION TITLE
I want to support leaderboard of bot on lishogi, lidraughts and playstrategy

### DISCUSSION THREAD

**Discussion by M-DinhHoangViet** [TheYoBots/libot-lb#22] [Ideas]

**Comment by TheYoBots** [comment]
I don't think those sites have many bots eligible for the leaderboard. In lishogi for example 0 bots are eligible.

**Comment by M-DinhHoangViet** [comment]
So how can I do the same thing without https://github.com/lichess-org/berserk?

**Comment by TheYoBots** [comment]
This repository doesn't use [berserk](https://pypi.org/project/berserk/). Although if I use it the code might be significantly reduced. I could use berserk if you want?

**Reply by M-DinhHoangViet** [reply]
I want it now, can you make it now?

**Reply by TheYoBots** [reply]
I don't think that this will make any sense since there are only few (or none) bots that are actually eligible to be in the leaderboard in these sites as I mentioned earlier.

**Comment by M-DinhHoangViet** [comment]
Ok, but lishogi and etc not have

**Comment by TheYoBots** [comment]
Yes, berserk is for lichess not for other sites. What exactly is it that you want me to do?

**Comment by M-DinhHoangViet** [comment]
I mean this code can use import lichess.api but can't use lihogi.api or playstrategy.api,..

**Comment by TheYoBots** [comment]
Yes, `import lichess.api` is from the package [python-lichess](https://pypi.org/project/python-lichess/) and there isn't a python package for lishogi/playstategy api.

**Comment by M-DinhHoangViet** [comment]
So can you follow my title?

**Comment by TheYoBots** [comment]
I'll have to think about this feature. Because there aren't too many bots on those sites and if I do implement this I feel almost all leaderboards will be empty.

**Comment by M-DinhHoangViet** [comment]
Ok

**Reply by TheYoBots** [reply]
https://github.com/TheYoBots/libot-lb/issues/21
